internal/store/ingress_test.go

```diff
@@ -169,7 +169,7 @@ func TestIngressStore(t *testing.T) {
 				kube_ingress_created{namespace="ns4",ingress="ingress4"} 1.501569018e+09
 				kube_ingress_labels{namespace="ns4",ingress="ingress4"} 1
 				kube_ingress_path{namespace="ns4",ingress="ingress4",host="somehost",path="/somepath",service_name="someservice",service_port="1234"} 1
-				kube_ingress_path{namespace="ns4",ingress="ingress4",host="somehost",path="/somepath2",service_name="",service_port=""} 1
```

Comment:
QQ: why service_name and service_port are deleted?

Reply:
@CatherineF-dev If the Ingress backend is of resource type, it will not have service_name, service_port. Resource & Service for a backend are mutually exclusive.
Ref: https://kubernetes.io/docs/concepts/services-networking/ingress/#resource-backend